src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -76,25 +68,38 @@ def test_core_class(device):
     assert np.allclose(results[list(results)[0]], expected_output)
 
 
-# request - https://docs.pytest.org/en/7.1.x/reference/reference.html#request
-def test_compile_model(request, tmp_path, device):
-    core = Core()
-    xml_path, bin_path = create_filename_for_test(request.node.name, tmp_path)
+@pytest.mark.parametrize("loading_type", [
+    None,
+    "posixpath",
+    "auto"
+])
+@pytest.mark.parametrize("device_name", [
+    None,
+    "CPU"
+])
+@pytest.mark.parametrize("config", [
+    None,
+    {hints.performance_mode(): hints.PerformanceMode.THROUGHPUT},
+    {hints.performance_mode(): hints.PerformanceMode.LATENCY},
+    {hints.performance_mode(): hints.PerformanceMode.CUMULATIVE_THROUGHPUT}
+])
+def test_compact_api(request, tmp_path, loading_type, device_name, config):
     relu_model = get_relu_model()
-    serialize(relu_model, xml_path, bin_path)
-    model = core.read_model(model=xml_path, weights=bin_path)
-    compiled_model = core.compile_model(model, device)
```

Comment:
please, revert changes here, as this test `test_compile_model` checked `core.compile_model` functionality.
move your changes to the test with name `test_compact_api_xml_posix_path`, which for sure may be renamed to `test_compact_api`